### PR TITLE
Strategy texts for units

### PR DESCRIPTION
**Pull request by MeuhV1**
Minor fixes

### CODE REVIEW

File: Assets/XML/Text/Units.xml [hunk 1]
```diff
@@ -920,7 +920,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_ARQUEBUSIER_STRATEGY</Tag>
-		<English>[COLOR_UNIT_TEXT]Musketmen[COLOR_REVERT] are the first [COLOR_UNIT_TEXT]Gunpowder Unit[COLOR_REVERT] avaiable.</English>
+		<English>[COLOR_UNIT_TEXT]Arquebusiers[COLOR_REVERT] are the first [COLOR_UNIT_TEXT]Gunpowder Unit[COLOR_REVERT] avaiable.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: available

File: Assets/XML/Text/Units.xml [hunk 2]
```diff
@@ -981,7 +981,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_PERSIAN_IMMORTAL_STRATEGY</Tag>
-		<English>The [COLOR_UNIT_TEXT]Immortal[COLOR_REVERT], the Persian replacement for the [COLOR_UNIT_TEXT]Spearman[COLOR_REVERT], has an attack bonus against [COLOR_UNIT_TEXT]Archers[COLOR_REVERT], is cheaper and begins with the [COLOR_HIGHLIGHT_TEXT]March[COLOR_REVERT] promotion.</English>
+		<English>The [COLOR_UNIT_TEXT]Immortal[COLOR_REVERT], the Persian replacement for the [COLOR_UNIT_TEXT]Spearman[COLOR_REVERT], has an attack bonus against [COLOR_UNIT_TEXT]Archer[COLOR_REVERT] units, is cheaper and begins with the [COLOR_HIGHLIGHT_TEXT]March[COLOR_REVERT] promotion.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: "Archer units" suggests a unit category, as in "Archery units".

File: Assets/XML/Text/Units.xml [hunk 3]
```diff
@@ -1029,15 +1029,15 @@
...
-		<English>The [COLOR_UNIT_TEXT]Legion[COLOR_REVERT], the Roman replacement for the the [COLOR_UNIT_TEXT]Swordsman[COLOR_REVERT], has two additional strength points, is more expensive, loses the attack bonus when attacking cities, has a combat bonus against the [COLOR_UNIT_TEXT]Axeman[COLOR_REVERT] unit, can construct [COLOR_HIGHLIGHT_TEXT]Roman Roads[COLOR_REVERT] and can be trained until the [COLOR_TECH_TEXT]Rifling[COLOR_REVERT] and [COLOR_TECH_TEXT]Chemistry[COLOR_REVERT] technologies have been discovered.</English>
+		<English>The [COLOR_UNIT_TEXT]Legion[COLOR_REVERT], the Roman replacement for the the [COLOR_UNIT_TEXT]Swordsman[COLOR_REVERT], has two additional strength points, is more expensive, has a combat bonus against the [COLOR_UNIT_TEXT]Axeman[COLOR_REVERT] unit and can construct [COLOR_HIGHLIGHT_TEXT]Roman Roads[COLOR_REVERT].</English>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_RIFLEMAN_STRATEGY</Tag>
-		<English>[COLOR_UNIT_TEXT]Riflemen[COLOR_REVERT] have a combat bonus against [COLOR_UNIT_TEXT]Mounted Units[COLOR_REVERT].</English>
+		<English>[COLOR_UNIT_TEXT]Riflemen[COLOR_REVERT] requires no special resources.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: That's kind of underwhelming. I would prefer something like "... is the basic gunpowder unit of the late Renaissance"

File: Assets/XML/Text/Units.xml [hunk 4]
```diff
@@ -1097,11 +1097,11 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_IRANIAN_QIZILBASH_STRATEGY</Tag>
-		<English>The [COLOR_UNIT_TEXT]Qizilbash[COLOR_REVERT], the Iranian replacement for the [COLOR_UNIT_TEXT]Musketman[COLOR_REVERT], has a combat bonus against [COLOR_UNIT_TEXT]Gunpowder Units[COLOR_REVERT] and begins with the [COLOR_HIGHLIGHT_TEXT]Guerilla I[COLOR_REVERT] promotion.</English>
+		<English>The [COLOR_UNIT_TEXT]Qizilbash[COLOR_REVERT], the Iranian replacement for the [COLOR_UNIT_TEXT]Arquebusier[COLOR_REVERT], has a combat bonus against [COLOR_UNIT_TEXT]Gunpowder Units[COLOR_REVERT] and begins with the [COLOR_HIGHLIGHT_TEXT]Guerilla I[COLOR_REVERT] promotion.</English>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_IRONCLAD_STRATEGY</Tag>
-		<English>[COLOR_UNIT_TEXT]Ironclads[COLOR_REVERT] can bombard city defenses and cannot enter [COLOR_HIGHLIGHT_TEXT]Ocean[COLOR_REVERT].</English>
+		<English>[COLOR_UNIT_TEXT]Ironclads[COLOR_REVERT] can bombard city defenses.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: Same here.

File: Assets/XML/Text/Units.xml [hunk 5]
```diff
@@ -1134,19 +1134,19 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_SHIP_OF_THE_LINE_STRATEGY</Tag>
-		<English>[COLOR_UNIT_TEXT]Ship of the Lines[COLOR_REVERT] can bombard city defenses and have a combat bonus against the [COLOR_UNIT_TEXT]Frigate[COLOR_REVERT] units.</English>
+		<English>[COLOR_UNIT_TEXT]Ship of the Lines[COLOR_REVERT] can bombard city defenses.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: And here.

File: Assets/XML/Text/Units.xml [hunk 6]
```diff
@@ -1134,19 +1134,19 @@
...
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_NUCLEAR_BOMBER_STRATEGY</Tag>
-		<English>[COLOR_UNIT_TEXT]Tactical Nukes[COLOR_REVERT] have an increased chance to evade interception and can nuke enemy lands.</English>
+		<English>[COLOR_UNIT_TEXT]Nuclear Bombers[COLOR_REVERT] can nuke enemy lands.</English>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_SWORDSMAN_STRATEGY</Tag>
-		<English>[COLOR_UNIT_TEXT]Swordsmen[COLOR_REVERT] have a attack bonus when attacking cities.</English>
+		<English>[COLOR_UNIT_TEXT]Swordsmen[COLOR_REVERT] have no special bonus.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: And here.

File: Assets/XML/Text/Units.xml [hunk 7]
```diff
@@ -1449,30 +1445,58 @@
...
+		<Tag>TXT_KEY_UNIT_TORPEDO_BOAT_STRATEGY</Tag>
+		<English>[COLOR_UNIT_TEXT]Torpedo Boats[COLOR_REVERT] cannot enter [COLOR_HIGHLIGHT_TEXT]Ocean[COLOR_REVERT], have a first strike, can withdraw from combat and have a combat bonus attacking on [COLOR_HIGHLIGHT_TEXT]Coast[COLOR_REVERT].</English>
+	</TEXT>
+	<TEXT>
 		<Tag>TXT_KEY_UNIT_CRUISER</Tag>
 		<English>Cruiser</English>
 	</TEXT>
 	<TEXT>
+		<Tag>TXT_KEY_UNIT_CRUISER_STRATEGY</Tag>
+		<English>[COLOR_UNIT_TEXT]Cruisers[COLOR_REVERT] can bombard city defenses.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: And here.

File: Assets/XML/Text/Units.xml [hunk 8]
```diff
@@ -1489,27 +1513,59 @@
...
+		<Tag>TXT_KEY_UNIT_HOWITZER_STRATEGY</Tag>
+		<English>[COLOR_UNIT_TEXT]Howitzers[COLOR_REVERT] cannot capture enemy cities or units, don't receive defensive bonuses, have a combat bonus when attacking cities, cause collateral damage, are immune to collateral damage from [COLOR_UNIT_TEXT]Siege Weapons[COLOR_REVERT] and can bombard city defenses.</English>
+	</TEXT>
+	<TEXT>
 		<Tag>TXT_KEY_UNIT_DRAGOON</Tag>
 		<English>Dragoon</English>
 	</TEXT>
 	<TEXT>
+		<Tag>TXT_KEY_UNIT_DRAGOON_STRATEGY</Tag>
+		<English>[COLOR_UNIT_TEXT]Dragoons[COLOR_REVERT] don't receive defensive bonuses.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: And here.

File: Assets/XML/Units/CIV4UnitInfos.xml [hunk 9]
```diff
@@ -566,7 +567,7 @@
 			<Domain>DOMAIN_LAND</Domain>
 			<DefaultUnitAI>UNITAI_WORKER</DefaultUnitAI>
 			<Description>TXT_KEY_UNIT_LABOURER</Description>
-			<Civilopedia>TXT_KEY_UNIT_LABOURER_PEDIA</Civilopedia>
+			<Civilopedia>TXT_KEY_UNIT_WORKER_PEDIA</Civilopedia>
```
Comment (dguenms) [inline Assets/XML/Units/CIV4UnitInfos.xml]: Please copy the worker pedia instead to use the LABOURER_PEDIA tag.

File: Assets/XML/Units/CIV4UnitInfos.xml [hunk 10]
```diff
@@ -1480,7 +1481,7 @@
 			<Domain>DOMAIN_LAND</Domain>
 			<DefaultUnitAI>UNITAI_ATTACK</DefaultUnitAI>
 			<Description>TXT_KEY_UNIT_MILITIA</Description>
-			<Civilopedia>TXT_KEY_UNIT_MILITIA_PEDIA</Civilopedia>
+			<Civilopedia>TXT_KEY_UNIT_WARRIOR_PEDIA</Civilopedia>
```
Comment (dguenms) [inline Assets/XML/Units/CIV4UnitInfos.xml]: Militia shouldn't use the Warrior pedia entry.

**Review by dguenms — CHANGES_REQUESTED** [review]

**Comment by MeuhV1** [comment]
Applied comments.

**Review by dguenms — APPROVED** [review]
Looks good!

**Pull request merged**